Codeinwp/neve
# fix: block preview not working properly in editor sidebar styles [#1094]

### Summary
Block Preview inside the editor Sidebar's Style accordion wasn't showing up because of previous changes made to the Gutenberg integration.


### Will affect visual aspect of the product

NO

### Screenshots 

Fixed
---
![image](https://user-images.githubusercontent.com/15010186/70725452-11dd9e00-1d05-11ea-9627-9fc5f2208526.png)

Previously
---
![image](https://user-images.githubusercontent.com/15010186/70724974-41d87180-1d04-11ea-926b-6e995640d933.png)

### Test instructions


- Check if the preview works properly now in the editor Styles accordion
- Check if the button styles are properly applied in the editor and on the front end


Refers to #1094.
